scripts/check-time-drift.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
This file was added to git with 664 perms which means it's not executable by the time it's bundled into inspector and delivered to a customer site.  It should have 755 perms. Just chmod it and then `git add` it, and that should fix it.